src/conv/k-g/region_list.cpp

```diff
@@ -49,16 +49,60 @@ Bot& RegionList::addRegion
 }
 
 
+void RegionList::setAttributes
+(
+    const std::string&                        name,
+    const std::map<std::string, std::string>& attributes
+) {
+    m_list[name].attributes = attributes;
+}
+
+static void  writeAttributes
+(
+    rt_wdb* wdbp,
+    const char* name,
+    const std::map<std::string, std::string>& attributes
+) {
+
+    struct rt_db_internal     region_internal;
+
+    struct db_i*      dbip = wdbp->dbip;
+    struct directory* dp = db_lookup(dbip, name, 0);
+
+    if (dp == RT_DIR_NULL) {
+	bu_log("writeAttributes() Cannot find %s\n", name);
+	return;
+    }
+
+    if (rt_db_get_internal(&region_internal, dp, dbip, NULL, &rt_uniresource) > 0) {

```

Comment:
**>= 0**?
BTW, the documentation I referred to where the comments in the header file include/rt/db_internal.h.

Hmm, it should be in the Doxygen documentation https://brlcad.org/docs/api/ (could be a bit outdated)
https://brlcad.org/docs/api/d2/de5/db__internal_8h.html#a5fabf39a9c6246df52172a92c2a2b221

This are the kind of references you can find only if you know that they exist :smiling_imp: